user_docs/en/changes.t2t

```diff
@@ -7,12 +7,12 @@ What's New in NVDA
 This release includes optional experimental support for UIA in Excel and Chromium browsers.
 There are fixes for several languages, and for accessing links in Braille.
 There are updates to Unicode CLDR and mathematical symbols, eSpeak-NG and LibLouis.
-
 Many bug fixes and improvements, including in Office, Visual Studio and several languages.
-Note: this is an Add-on API compatibility breaking release.
-Add-ons will need to be re-tested and have their manifest updated.
 This release also drops support for Adobe Flash.
 
+Note: this is an Add-on API compatibility breaking release.
+Add-ons will need to be re-tested and authors will need to provide a new version certifying its compatibility with this release.

```

Comment:
```suggestion
Add-ons will need to be re-tested.
Add-on authors will need to release a new version with an updated manifest to certify compatibility with this release.
```

Reply:
This is tricky, as a message to add-on authors this isn't detailed enough. It could say consult the "changes for developers" section and check if your addon relies on any of the changed / moved / removed code. However, for regular users that is way too technical for a release blurb / highlight.

Perhaps we could just add "consult the changes for developers".

Reply:
Thoughts on the following?

```suggestion
Add-ons will need to be re-tested and release a new version which certifies compatibility with this version of NVDA.
Add-on authors should consult the changes for developers.
```

Reply:
To clarify my intention when rewording this sentence:
1. I have assumed that the blurb was for regular users and that add-on developers know that they need to look at the change for developers. Thus,
2. I absolutely wanted to remove the word manifest in the blurb since
  * the regular user does not now what it is about
  * or much worse the regular user may be tempted to modify himself the manifest in the place of the add-on author, what is not recommended at all; we can see regularly such messages on the mailing lists

Regarding the the following sentence:
> Add-on authors should consult the changes for developers.
I personally do not think it is required since the blurb is already there as an incitation for people to read the rest of the change log. Let me know if you want me to add it anyway.

Reply:
Here is a suggestion. Feel free to comment:
Even if it targets add-on authors, regular users should understand that they need to wait for an updated add-on release.

```suggestion
Add-on authors will need to re-tested their add-on and authors will need to provide a new version certifying its compatibility with this release.

```

Reply:
```suggestion
Add-ons will need to re-tested.
Add-on authors will need to provide a new version certifying its compatibility with this release.
Add-on authors should use the "changes for developers" to guide their update.
```

Reply:
Still refining the formulation.
From @feerrenrut's suggestion:
* added the missing word "be"
* replaced the second occurrence of "Add-on authors" with "They"
Hope it's OK now!

```suggestion
Add-ons will need to be re-tested.
Add-on authors will need to provide a new version certifying its compatibility with this release.
They should use the "changes for developers" to guide their update.

```

Reply:
Whatever is done here, please get the plurals right. If saying add-ons, or add-on authors, it is awkward to then talk about the add-on in the singular ("it", "a new version").

```suggestion
Add-on authors will need to test their add-ons with this release, and provide new versions certifying their compatibility.
Authors should see the "Changes for Developers" section of the changes file for details.
```

Alternative suggestion:

```suggestion
Add-on authors will need to test their add-ons with 2021.1, and release new add-on versions which declare compatibility with it.
Authors should see the "Changes for Developers" section of the changes file for details.
```

Reply:
There are many comments for these sentence!

Personally, I would be willing to accept @XLTechie's first suggestion, i.e.: 
> Add-on authors will need to test their add-ons with this release, and provide new versions certifying their compatibility.
> Authors should see the "Changes for Developers" section of the changes file for details.
especially for the first sentence.

I prefer this formulation for the following points:
* It indicates clearly who is in charge of add-on re-testing
* It indicates clearly the nature and the objective of the tests, i.e. check and certify compatibility with this release.
* No mention of the manifest in the blurb to avoid the temptation for regular users to modify it
* No singular / plural mismatch
* Reference to the "Changes for dev" section; from the last comments, it seemed a consensus to have this reference.

Please for anybody, if you make an alternative suggestion, indicate your motivation with the points above and new ones. Thanks.

Reply:
Since the last time an update broke add-ons was 2019.3, I went back to the blurb for that update, which says simply:

(Point about update to Python and speech subsystem)... Although these changes do break compatibility with older NVDA add-ons, the upgrade to Python 3 ... (highlighting importance of updates).

Working from that, perhaps simply:

Note: This release breaks compatibility with existing add-ons
```suggestion
Note: This release breaks compatibility with existing add-ons

```
I contemplated whether to make it "Note: This release breaks compatibility with existing add-ons, which will need to be updated.", but I'm not convinced that latter part needs to be there - existing add-ons are broken, therefore they need to be updated.


As noted.  Add-on authors should be familiar with reading the changes for developers section.  I don't think we should need to specify that they read that section.  Since we are already saying it breaks add-ons, it should follow that the extra detail on why should be contained in that changes for developers section.

Reply:
I think the idea when telling that add-on authors should provide new versions of their add-ons is to avoid questions or issues such as #12426. So IMO specifying it remains important.

Anyway, I think everyone has been able to write their arguments on how to write this sentence. At the end, it's up to NVAccess people to decide.
@feerrenrut, @seanbudd, @Qchristensen for now there is no converging proposal from you three. Could you try to agree on what is important (re-reading the whole thread) or indicate who of you three is in charge to decide/validate at the end? Thanks.

Reply:
Yep, let's go with @Qchristensen's suggestion. However, I think we should use a list to have discreet items for the "Note", including the flash item.